test/e2e/apibinding/apibinding_authorizer_test.go

```diff
@@ -251,7 +269,20 @@ func testCRUDOperations(ctx context.Context, t *testing.T, consumer1Workspace lo
 			Namespace: "default",
 		},
 	}
-	_, err := cowboyClient.Create(ctx, cowboy, metav1.CreateOptions{})
+	_, err := cowboyClient.Create(kcp.WithCluster(ctx, consumer1Workspace), cowboy, metav1.CreateOptions{})
 	require.NoError(t, err, "error creating cowboy in consumer workspace %q", consumer1Workspace)
 
 }
+
+// getKcpClient modifies the provided config's hose to point towards the given cluster
+// and returns a client to access objects in that cluster.
+func getKcpClient(config *rest.Config, clusterName logicalcluster.Name) (*clientset.Clientset, error) {
```

Comment:
Would be nice to move this helper in a common place. This is frequently used for creating modified kcp clients. We need a similar one for k8s client too.

Till we figure out a way to scope discovery client calls, would it be reasonable to scaffold this helper for custom clients for users?

Reply:
Done.